docs/machine-learning/tutorials/health-violation-classification-model-builder.md

```diff
@@ -0,0 +1,169 @@
+1. Select the button next to the **Connect to SQL Server database** text box.
+    1. In the **Choose Data** dialog, select **Microsoft SQL Server Database File**. 
+    1. Uncheck the **Always use this selection** checkbox and select **Continue**
+    1. In the **Connection Properties** dialog, select **Browse** and select the downloaded *RestaurantScores.mdf* file.
+    1. Select **OK**
+1. Choose **Violations** from the **Table Name** dropdown.
+1. Choose **RiskCategory** in the **Column to Predict (Label)** dropdown.
+1. Leave the default column selections, **InspectionType** and **ViolationDescription**, checked in the **Input Columns (Features)** dropdown.
+1. Select the **Train** link to move to the next step in the Model Builder tool.
+
+## Train the model
+
+The machine learning task used to train the issue classification model in this tutorial is multiclass classification. During the model training process, Model Builder trains separate models using different multiclass classification algorithms and settings to find the best performing model for your dataset.
+
+The time required for the model to train is proportional to the amount of data. Model Builder automatically selects a default value for **Time to train (seconds)** based on the size of your data source.
+
+1. Although Model Builder sets the value of **Time to train (seconds)** to 10 seconds, increase it to 30 seconds. Training for a longer period of time allows Model Builder to explore a larger number of algorithms and combination of parameters in search of the best model.
+1. Select **Start Training**.
+
+    Throughout the training process, progress data is displayed in the `Progress` section of the train step.
+
+    - Status displays the completion status of the training process.
+    - Best accuracy displays the accuracy of the best performing model found by Model Builder so far. Higher accuracy means the model predicted more correctly on test data.
+    - Best algorithm displays the name of the best performing algorithm performed found by Model Builder so far.
+    - Last algorithm displays the name of the algorithm most recently used by Model Builder to train the model.
+
+1. Once training is complete, select the **evaluate** link to move to the next step.
+
+## Evaluate the model
+
+The result of the training step will be one model, which had the best performance. In the evaluate step of the Model Builder tool, the output section, will contain the algorithm used by the best performing model in the **Best Model** entry along with metrics in **Best Model Accuracy**. Additionally, a summary table containing top five models and their metrics.
+
+If you're not satisfied with your accuracy metrics, some easy ways to try to improve model accuracy are to increase the amount of time to train the model or use more data. Otherwise, select the **code** link to move to the final step in the Model Builder tool.
```

Comment:
```suggestion
If you're not satisfied with your accuracy metrics, some easy ways to try to improve model accuracy are to increase the amount of time to train the model or use more data. Otherwise, select the **code** link to move to the final step in Model Builder.
```